test-cases/tests/installation/a30-validate-installation-of-rhoam-addon-and-integration-with-ld.md

```diff
@@ -51,6 +51,14 @@ Host prefix: /26
 
 9. Click on `Create cluster` (cluster creation takes ~40 minutes)
 
+Run following command:
+
+```
+ocm get subs --parameter search="cluster_id = '<your-cluster-ID>'" | jq -r .items[0].metrics[0].health_state
+```
+
+Provide your cluster ID (e.g. 1lblvj4k7qgsqimqhbkc64n9q350748n). The command should return `healthy`.
```

Comment:
```suggestion
The command should return `healthy`.
```
(Please apply this if you agree with the suggestion above).
Also would it be worth to add an advice on what to do in case the cluster is not healthy?

Reply:
It would, but I have no piece of advice. The cluster should get `healthy` eventually. If not, the only thing I can think of is to try again